### PR TITLE
add creating Image model

### PR DESCRIPTION
**Pull request by tsyboom**

### CODE REVIEW

File: wsgi.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def update_vidaxl_products():
     """Update all products from VidaXl
     """
     from app.controllers import download_products
-    download_products()
+    download_products(10000)
```
Comment (kipod) [inline wsgi.py]: pls revert

File: app/controllers/products.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def in_selected_category(category_path):
                             ],
                             images=[
                                 {"src": img}
-                                for img in scrap_img(product.vidaxl_id).get(
+                                for img in scrap_img(product.vidaxl_id, product.id).get(
```
Comment (kipod) [inline app/controllers/products.py:201]:
```python
scrap_img(product)
```

**Review by kipod — CHANGES_REQUESTED** [review]

File: wsgi.py (hunk 1 above)
Comment (tsyboom) [reply wsgi.py]: done

File: app/controllers/products.py (hunk 2 above)
Comment (tsyboom) [reply app/controllers/products.py:201]: done

**Review by kipod — APPROVED** [review]

**Comment by kipod** [comment]
where unit tests?

**Pull request merged**